### DISCUSSION TITLE
How to connect to from MPD to icecast over SSL/HTTPS

### DISCUSSION THREAD

**Discussion by tbharber** [MusicPlayerDaemon/MPD#1278] [Q&A]
Hey All,

Just converted my Icecast and domain to use a SSL cert.  I can browse to the status page just fine via the browser, but MPD is now failing connection.  When it tries to connect I get the following in the logs:
`xception: nested: problem opening connection to shout server localhost:8000: Socket error`

Does anyone know what configuration steps need to be taken to get MPD to talk to Icecast using SSL?

**Comment by MaxKellermann** [comment]
I've never used Icecast (except for debugging the MPD plugin), but if you want help, you should post more information. See https://www.musicpd.org/help/
